apps/meteor/ee/server/models/raw/LivechatDepartment.ts

```diff
@@ -75,11 +77,13 @@ export class LivechatDepartmentEE extends LivechatDepartmentRaw implements ILive
 	async findEnabledWithAgentsAndBusinessUnit<T extends Document = ILivechatDepartment>(
 		businessUnit: string,
 		projection: FindOptions<T>['projection'],
+		extra?: Record<string, any>,
```

Comment:
is this param that open? I've only seen it being used to pass a `{userId}` object, could we make the types more strict?